jenkins/bisect.jpl

```diff
@@ -538,8 +542,28 @@ def bisection(kci_core, kdir, checks) {
             )
         }
 
+        dir(kci_core) {
+            if (!(ref_kernel_url && ref_kernel_tree && params.ref_kernel_branch)) {
+
+                ref = sh(script: """\
+                                ./kci_build get_reference --tree-name ${params.KERNEL_TREE} \
+                                --branch ${params.KERNEL_BRANCH}""", returnStdout: true).trim().tokenize("\n")
+
+                if (ref.size() > 0) {
+                    ref_kernel_url = ref[0]
+                    ref_kernel_tree = ref[1]
+                    ref_kernel_branch = ref[2]
+                }
+            }
```

Comment:
This code should be moved to `findMergeBase()` and only run if the good revision is not already a merge base.

Reply:
Corrected